src/main/kotlin/ResponseBuilder.kt

```diff
@@ -1,16 +1,17 @@
 package response
 
-class ResponseBuilder(private val statusCode: Int, private val body: String = "") {
+class ResponseBuilder(private val statusCode: Int,
+                      private val body: String = "",
+                      private val headers: Map<String, List<String>> = mapOf()
```

Comment:
Looks like a header always has to be a `List<String>`. Is it possible that it's just a string or even a number? If so, would we need to convert them into a list of strings?

Reply:
Thanks for pointing this out! A header value can be a string or even a number. I'm not sure if I should wait until the change is necessary or go ahead and change the type to maybe `Any`? I don't think I'll run into any problems if I do this, because in the `generateHeaders` method I am joining the `List` to a `String`. However, declaring the `List` as a type of `Any` increases the risk of adding non-string and non-number types to the `List`.